plugin/trino-hive/src/main/java/io/trino/plugin/hive/metastore/HiveMetastore.java

```diff
@@ -62,10 +62,20 @@ default void updatePartitionStatistics(Table table, String partitionName, Functi
 
     List<String> getAllTables(String databaseName);
 
+    /**
+     * @return List of table names from all schemas or Optional.empty if operation is not supported
```

Comment:
Lists tables, views and materialized views

Reply:
Done